src/marqo/client.py

```diff
@@ -195,7 +195,14 @@ def health(self):
         mq_logger.warning('The `client.health()` API has been deprecated and will be removed in '
                           'Marqo 2.0.0. Use `client.index(index_name).health()` instead. '
                           'Check `https://docs.marqo.ai/latest/API-Reference/indexes/` for more details.')
-        return self.http.get(path="health")
+        try:
+            return self.http.get(path="health")
+        except (MarqoWebError, RequestException, TypeError, KeyError) as e:
+            raise errors.BadRequestError("Marqo encountered an error trying to check the health of the Marqo instance. "
+                                         "It is likely you are trying to check the health on Marqo Cloud V2. "
```

Comment:
Let's make less assumptions in our language: 
`"It is likely you are trying to check the health` -> `If you are trying to check the health`